crates/test-helpers/build.rs

```diff
@@ -149,6 +151,24 @@ fn main() {
                 test_dir.file_stem().unwrap().to_str().unwrap().to_string(),
                 out_wasm.to_str().unwrap().to_string(),
             ));
+
+            // The "invalid" test doesn't actually use the rust-guest macro
+            // and doesn't put the custom sections in, so component translation
+            // will fail.
+            if test_dir.file_stem().unwrap().to_str().unwrap() != "invalid" {
```

Comment:
I think with my suggestion above about how to force linking the object file it should resolve this?

Reply:
It turns out to be unnecessary now even without that fix, I just forgot to try it again